igvm/hypervisor.py

```diff
@@ -57,12 +57,27 @@ def __init__(self, dataset_obj):
         # We cannot store these in the VM object due to migrations.
         self._mount_path = {}
 
-    def vm_disk_path(self, name):
-        return '/dev/{}/{}'.format(VG_NAME, name)
+    def vm_lv_get(self, vm):
```

Comment:
We name most of our functions as normal English sentences would go, e.g. "get_vlan_this", "format_vm_that", "download_those". Please stick to the rule.